covid19-etl/etl/idph_hospital.py

```diff
@@ -0,0 +1,278 @@
+            "TotalBedsUsed": "state_total_beds_used",
+            "PctHospitalBedsAvailable": "state_pct_hospital_beds_available",
+            "AdultICUCapacity": "state_adult_icu_capacity",
+            "ICUOpenBeds": "state_icu_open_beds",
+            "ICUBedsUsed": "state_icu_beds_used",
+            "ICUOpenBedsPct": "state_icu_open_beds_pct",
+            "COVIDPUIPatients": "state_covid_pui_patients",
+            "COVIDPUIPatientsPct": "state_covid_pui_patients_pct",
+            "COVIDPUIPatientsBedsInUsePct": "state_covid_pui_patients_beds_in_use_pct",
+            "VentilatorCapacity": "state_ventilator_capacity",
+            "VentilatorsOpen": "state_ventilators_open",
+            "VentilatorsOpenPct": "state_Ventilators_open_pct",
+            "VentilatorsInUse": "state_ventilators_in_use",
+            "VentilatorsInUseCOVID": "state_ventilators_in_use_covid",
+            "VentilatorsCOVIDPatientsPct": "state_ventilators_covid_patients_pct",
+            "VentilatorsCOVIDPatientsInUsePct": "state_ventilators_covid_patients_in_use_pct",
+            "CovidPatientsNonICU": "state_covid_patients_non_icu",
+            "TotalCOVIDPUIInICU": "state_total_covid_pui_in_icu",
+            "TotalCOVIDPUIInHospital": "state_total_covid_pui_in_hospital",
+            "PctBedsCOVIDPUI": "state_pct_beds_covid_pui",
+            "MedSurgBeds": "state_med_surg_beds",
+            "MedSurgBedsOpen": "state_med_surg_beds_open",
+            "MedSurgBedsOpenPct": "state_med_surg_beds_open_pct",
+            "MedSurgBedsInUse": "state_med_surg_beds_in_use",
+        }
+
+        summary_location_submitter_id = format_submitter_id(
+            "summary_location",
+            {
+                "project": "idph_hospital",
+                "country": self.country,
+                "state": self.state,
+            },
+        )
+
+        summary_location = {
+            "submitter_id": summary_location_submitter_id,
+            "projects": [{"code": self.project_code}],
+            "country_region": self.country,
+            "province_state": self.state,
+        }
+
+        summary_clinical_submitter_id = derived_submitter_id(
+            summary_location_submitter_id,
+            "summary_location",
+            "summary_clinical",
+            {"project": "idph_hospital", "date": date},
+        )
+
+        summary_clinical = {
+            "submitter_id": summary_clinical_submitter_id,
+            "date": date,
+            "summary_locations": [{"submitter_id": summary_location_submitter_id}],
+        }
+
+        for k, v in statewide_values.items():
+            summary_clinical[statewide_mapping[k]] = v
+
+        return summary_location, summary_clinical
+
+    def parse_region(self, date, hospital_region):
+        """
+        From county-level data, generate the data we can submit via Sheepdog
+        """
+        region = hospital_region["region"]
+        region_id = hospital_region["id"]
```

Comment:
this is not used, is it an oversight?

Reply:
`region_id` is not used. It makes more sense to use whole region identifier from `region`. I'll remove it.